modules/devices/x86/processor.c

```diff
@@ -610,6 +660,10 @@ gchar *processor_get_detailed_info(Processor * processor)
 			  "Frequency=%.2fMHz\n"
 			  "BogoMIPS=%.2f\n"
 			  "Byte Order=%s\n"
+			  "[Freq]\n"
```

Comment:
I'd prefer if this were not abbreviated: "Frequency", "Minimum", "Maximum", and "Current" are more consistent.